### PR TITLE
Addition of missing smart descriptions

### PR DESCRIPTION
**Pull request by LenaigKaliou**
Fix to update several existing formats missing smart descriptions

### CODE REVIEW

File: Azure/azure-linux/tests/auth_conversation_failed.json [hunk 1]
```diff
@@ -16,9 +16,6 @@
     "azure_linux": {
       "message": "pam_unix(sudo:auth): conversation failed"
     },
-    "host": {
```
Comment (TOUFIKIzakarya) [inline Azure/azure-linux/tests/auth_conversation_failed.json:19]: I'm wondering why `host.name` was deleted ?

**Review by TOUFIKIzakarya — COMMENTED** [review]
Good job. 
I have just a small comment regarding `host.name` field

File: Azure/azure-linux/tests/auth_conversation_failed.json (hunk 1 above)
Comment (LenaigKaliou) [reply Azure/azure-linux/tests/auth_conversation_failed.json:19]: It is due to a previous fix on the format, the field is nested under log.hostname, the tests had not been updated

**Review by TOUFIKIzakarya — APPROVED** [review]
LGFM

**Pull request merged**